docker-compose.dev.yml

```diff
@@ -0,0 +1,65 @@
+services:
+  web:
+    container_name: web
+    image: ghcr.io/vim/website_next_generation/web:${TAG:-development}
```

Comment:
Does this work with the development branch?

Reply:
Yes, if TAG is not set, it will resolve to the default value of `development`.

Reply:
Okay so the `-` as prefix won't be recognized?

Reply:
Yes the dash indicates the default value, its just bash syntax